### PR TITLE
Display system, system version, and endpoint label as part of conduct ACL command

### PR DESCRIPTION
**Pull request by fsat**
Also provide warning message and some hints on how to fix the situation  when duplicate endpoint is encountered within a particular system and system version.

### CODE REVIEW

**Comment by fsat** [comment]
Manual tests completed successfully. 
- `sandbox run 0.1.0 -f logging`
- `export CONDUCTR_API_VERSION=2` - set to the version 2 of the API so the `systemVersion` property is being read from the `systemVersion` field instead of being derived from the value of `system` property
- Load 2 bundles which has duplicate endpoint on both HTTP and TCP endpoints
- Running `conduct acls` should display warning on these endpoints

Bundle conf:

```
name := "path-tester"

BundleKeys.bundleConfVersion := BundleConfVersions.V_1_2_0
BundleKeys.compatibilityVersion := "1.0"
BundleKeys.nrOfCpus := 1.0
BundleKeys.memory := 64.MiB
BundleKeys.diskSpace := 35.MB
BundleKeys.roles := Set("web")
javaOptions in Bundle ++= Seq("-Dhttp.address=$PTEST_BIND_IP", "-Dhttp.port=$PTEST_BIND_PORT")
BundleKeys.endpoints := Map(
  "ptest" -> Endpoint("http", 0, "path-tester",
    RequestAcl(
      Http(
        "^/bacon".r -> "/burger",
        "^/tree".r,
        "^/fee/(.*)/fi/(.*)/fo/(.*)/fum$".r -> "/boom/\\1-\\2-\\3/box",
        "^/boom/(.*)/box$".r,
        "/foo" -> "/baz",
        "/baz"
      )
    )
  ),
  "ptunnel" -> Endpoint("tcp", 0, "ptunnel",
    RequestAcl(
      Tcp(
        12101, 3303, 5601
      )
    )
  )
)
```

Other bundle conf which has duplicate endpoint:

```
name := "path-tester-dup"

BundleKeys.bundleConfVersion := BundleConfVersions.V_1_2_0
BundleKeys.system := "path-tester"
BundleKeys.compatibilityVersion := "1.0"
BundleKeys.nrOfCpus := 1.0
BundleKeys.memory := 64.MiB
BundleKeys.diskSpace := 35.MB
BundleKeys.roles := Set("web")
javaOptions in Bundle ++= Seq("-Dhttp.address=$PTEST_BIND_IP", "-Dhttp.port=$PTEST_BIND_PORT")
BundleKeys.endpoints := Map(
  "ptest" -> Endpoint("http", 0, "path-tester",
    RequestAcl(
      Http(
        "^/bacon".r -> "/burger",
        "^/tree".r,
        "^/fee/(.*)/fi/(.*)/fo/(.*)/fum$".r -> "/boom/\\1-\\2-\\3/box",
        "^/boom/(.*)/box$".r,
        "/foo" -> "/baz",
        "/baz"
      )
    )
  ),
  "ptunnel" -> Endpoint("tcp", 0, "ptunnel",
    RequestAcl(
      Tcp(
        12101, 3303, 5601
      )
    )
  )
)
```

Running `conduct acls` for both http and tcp endpoints produces the expected warning:

```
Felixs-MBP:path-tester felixsatyaputra$ conduct acls http
METHOD  PATH                             REWRITE             SYSTEM           SYSTEM VERSION  ENDPOINT  BUNDLE ID  BUNDLE NAME      STATUS
*       ^/fee/(.*)/fi/(.*)/fo/(.*)/fum$  /boom/\1-\2-\3/box  path-tester      1.0             ptest     092e2bb    path-tester-dup  Running
*       ^/fee/(.*)/fi/(.*)/fo/(.*)/fum$  /boom/\1-\2-\3/box  path-tester      1.0             ptest     4d81c49    path-tester      Running
*       ^/boom/(.*)/box$                                     path-tester      1.0             ptest     092e2bb    path-tester-dup  Running
*       ^/boom/(.*)/box$                                     path-tester      1.0             ptest     4d81c49    path-tester      Running
*       ^/                                                   conductr-kibana  1               kibana    527b1f9    conductr-kibana  Running
*       ^/bacon                          /burger             path-tester      1.0             ptest     092e2bb    path-tester-dup  Running
*       ^/tree                                               path-tester      1.0             ptest     092e2bb    path-tester-dup  Running
*       ^/bacon                          /burger             path-tester      1.0             ptest     4d81c49    path-tester      Running
*       ^/tree                                               path-tester      1.0             ptest     4d81c49    path-tester      Running
*       /foo                             /baz                path-tester      1.0             ptest     092e2bb    path-tester-dup  Running
*       /baz                                                 path-tester      1.0             ptest     092e2bb    path-tester-dup  Running
*       /foo                             /baz                path-tester      1.0             ptest     4d81c49    path-tester      Running
*       /baz                                                 path-tester      1.0             ptest     4d81c49    path-tester      Running

Warning: Multiple endpoints found: path-tester/1.0/ptest
Warning: Unable to generate proxy configuration for these endpoints.
```

```
Felixs-MBP:path-tester felixsatyaputra$ conduct acls tcp
TCP/PORT  SYSTEM       SYSTEM VERSION  ENDPOINT  BUNDLE ID  BUNDLE NAME      STATUS
3303      path-tester  1.0             ptunnel   092e2bb    path-tester-dup  Running
3303      path-tester  1.0             ptunnel   4d81c49    path-tester      Running
5601      path-tester  1.0             ptunnel   092e2bb    path-tester-dup  Running
5601      path-tester  1.0             ptunnel   4d81c49    path-tester      Running
12101     path-tester  1.0             ptunnel   092e2bb    path-tester-dup  Running
12101     path-tester  1.0             ptunnel   4d81c49    path-tester      Running

Warning: Multiple endpoints found: path-tester/1.0/ptunnel
Warning: Unable to generate proxy configuration for these endpoints.
```

File: conductr_cli/conduct_acls.py [hunk 1]
```diff
@@ -77,10 +92,19 @@ def tcp_port_value(line):
...
+            '{endpoint_name: <{endpoint_name_width}}{padding}'
             '{bundle_id: <{bundle_id_width}}{padding}'
             '{bundle_name: <{bundle_name_width}}{padding}'
             '{status: <{status_width}}'.format(**dict(row, **column_widths)).rstrip())
 
+    duplicate_endpoints = find_duplicate_endpoints(tcp_acls)
+    if duplicate_endpoints:
+        log.screen('')
+        log.warning('Multiple endpoints found: {}'.format(', '.join(duplicate_endpoints)))
+        log.warning('Unable to generate proxy configuration for these endpoints.')
```
Comment (huntc) [inline conductr_cli/conduct_acls.py]: Is this the right message? Are we attempting to generate proxy config?

**Comment by huntc** [comment]
LGTM - just one comment on the error message, otherwise ship it!

File: conductr_cli/conduct_acls.py (hunk 1 above)
Comment (fsat) [inline conductr_cli/conduct_acls.py]: If the endpoint is duplicated, we won't expose the endpoint through the proxy. Maybe `Unable to expose these endpoint via ConductR HAProxy` is a better error message?

Comment (huntc) [inline conductr_cli/conduct_acls.py]: Yes, I think so.

Comment (huntc) [inline conductr_cli/conduct_acls.py]: ...and also an explanation to the user on how to fix the problem.

**Comment by fsat** [comment]
@huntc - error message updated as below:

```
Felixs-MBP:path-tester felixsatyaputra$ conduct acls http
METHOD  PATH                             REWRITE             SYSTEM           SYSTEM VERSION  ENDPOINT NAME  BUNDLE ID  BUNDLE NAME      STATUS
*       ^/fee/(.*)/fi/(.*)/fo/(.*)/fum$  /boom/\1-\2-\3/box  path-tester      1.0             ptest          092e2bb    path-tester-dup  Running
*       ^/fee/(.*)/fi/(.*)/fo/(.*)/fum$  /boom/\1-\2-\3/box  path-tester      1.0             ptest          4d81c49    path-tester      Running
*       ^/boom/(.*)/box$                                     path-tester      1.0             ptest          092e2bb    path-tester-dup  Running
*       ^/boom/(.*)/box$                                     path-tester      1.0             ptest          4d81c49    path-tester      Running
*       ^/                                                   conductr-kibana  1               kibana         20d5d6d    conductr-kibana  Running
*       ^/bacon                          /burger             path-tester      1.0             ptest          092e2bb    path-tester-dup  Running
*       ^/tree                                               path-tester      1.0             ptest          092e2bb    path-tester-dup  Running
*       ^/bacon                          /burger             path-tester      1.0             ptest          4d81c49    path-tester      Running
*       ^/tree                                               path-tester      1.0             ptest          4d81c49    path-tester      Running
*       /foo                             /baz                path-tester      1.0             ptest          092e2bb    path-tester-dup  Running
*       /baz                                                 path-tester      1.0             ptest          092e2bb    path-tester-dup  Running
*       /foo                             /baz                path-tester      1.0             ptest          4d81c49    path-tester      Running
*       /baz                                                 path-tester      1.0             ptest          4d81c49    path-tester      Running

Warning: Multiple endpoints found: path-tester/1.0/ptest
Warning: Unable to expose these endpoint via ConductR HAProxy.
Warning: Please ensure the ENDPOINT NAME is unique within a particular SYSTEM and SYSTEM VERSION.
```

```
Felixs-MBP:path-tester felixsatyaputra$ conduct acls tcp
TCP/PORT  SYSTEM       SYSTEM VERSION  ENDPOINT NAME  BUNDLE ID  BUNDLE NAME      STATUS
3303      path-tester  1.0             ptunnel        092e2bb    path-tester-dup  Running
3303      path-tester  1.0             ptunnel        4d81c49    path-tester      Running
5601      path-tester  1.0             ptunnel        092e2bb    path-tester-dup  Running
5601      path-tester  1.0             ptunnel        4d81c49    path-tester      Running
12101     path-tester  1.0             ptunnel        092e2bb    path-tester-dup  Running
12101     path-tester  1.0             ptunnel        4d81c49    path-tester      Running

Warning: Multiple endpoints found: path-tester/1.0/ptunnel
Warning: Unable to expose these endpoint via ConductR HAProxy.
Warning: Please ensure the ENDPOINT NAME is unique within a particular SYSTEM and SYSTEM VERSION.
```

**Comment by huntc** [comment]
Nice

**Comment by fsat** [comment]
Thank you

**Pull request merged**